tests/FoodStorage/test_foodStorage.py

```diff
@@ -108,6 +109,14 @@ def testUpdateFoods(self):
 
         self.foodStorage.update(name1)
         self.assertEqual([food2, food1], self.foodStorage.list())
+    
+    def testAsDictionary(self):
+        actual = self.foodStorage.asDictionary() 
+        expected = {
+            EXPIRATION_WINDOW : 7,
+            FOOD : [],
+        } 
+        self.assertEqual(expected, actual)
         
```

Comment:
I think you should test the scenario for if there's food in the foodstorage

Reply:
Done, let me know what you think!